testing/termite/APKBUILD

```diff
@@ -0,0 +1,66 @@
+# Contributor: Ian Bashford <ianbashford@gmail.com>
+# Maintainer: Ian Bashford <ianbashford@gmail.com>
+pkgname=termite
+pkgver=15
+pkgrel=0
+_vtever=0.50.2-ng
+pkgdesc="A simple VTE-based terminal"
+url="https://github.com/thestinger/termite/"
+arch="x86_64"
+options="!check"  # No test suite.
+license="LGPL"
+depends="$pkgname-terminfo"
+makedepends="intltool libtool automake autoconf gtk-doc gtk+3.0-dev
+	glib-dev pcre2-dev gnutls-dev gperf ncurses
+	pcre2 gnutls"
+subpackages="$pkgname-doc $pkgname-terminfo::noarch"
+source="$pkgname-$pkgver.tar.gz::https://github.com/thestinger/termite/archive/v$pkgver.tar.gz
+        vte-ng-$_vtever.tar.gz::https://github.com/thestinger/vte-ng/archive/$_vtever.tar.gz"
+builddir="${srcdir}/${pkgname}-${pkgver}"
+
+unpack() {
+	default_unpack
+	cd "${pkgname}-${pkgver}"
+	git clone https://github.com/thestinger/util
```

Comment:
can this be added to source= instead ? 

See how telegram-desktop does it to add git-submodules

Reply:
Done.
Thanks for the pointer;  I was hoping to make that neater.